cli/cmd/specs.go

```diff
@@ -48,16 +48,16 @@ func CLISourceSpecToPbSpec(spec specs.Source) pbSpecs.Source {
 		Version:             spec.Version,
 		Path:                spec.Path,
 		Registry:            CLIRegistryToPbRegistry(spec.Registry),
-		Concurrency:         spec.Concurrency,
-		TableConcurrency:    spec.TableConcurrency,
-		ResourceConcurrency: spec.ResourceConcurrency,
+		Concurrency:         spec.Concurrency,         // nolint:staticcheck // allow use of deprecated options here for backwards-compatibility
```

Comment:
Maybe arrange the deprecated options down in the struct together... then the `allow use of...` comment can be at the top of the 'block'.